code/game/objects/items/weapons/cigs.dm

```diff
@@ -87,16 +88,19 @@ LIGHTERS ARE IN LIGHTERS.DM
 		var/obj/item/lighter/zippo/Z = I
 		if(Z.lit)
 			light("<span class='rose'>With a single flick of [user.p_their()] wrist, [user] smoothly lights [user.p_their()] [name] with [user.p_their()] [Z]. Damn [user.p_theyre()] cool.</span>")
+			playsound(src, 'sound/items/lighter/light.ogg', 25, TRUE)
 
 	else if(istype(I, /obj/item/lighter))
 		var/obj/item/lighter/L = I
 		if(L.lit)
 			light("<span class='notice'>After some fiddling, [user] manages to light [user.p_their()] [name] with [L].</span>")
+			playsound(src, 'sound/items/lighter/light.ogg', 25, TRUE)
```

Comment:
`playsound(src, 'sound/items/lighter/light.ogg', 25, TRUE)` should probably be added to the light() proc, otherwise you also need to to places like `else if(istype(I, /obj/item/assembly/igniter))` and welder_act

Reply:
Done